### ISSUE TITLE
Single container Helm chart

### ISSUE THREAD

**Issue by dabeeeenster** [Flagsmith/flagsmith-charts#95]
Default to the unified docker image

**Comment by matthewelwell** [comment]
From speaking to @plumdog, this is essentially done code-wise. If we want this to be the default behaviour, however, we just need to set the default value for `separateApiAndFrontend` in the `values.yaml` file to `false`. This will need a major version bump as it will be a breaking change for most people using the charts. 

So, the actions therefore are: 
 * I need to make sure the EE can function as a single container
 * We need to determine a place to have some notes about upgrading from one chart version to another when a breaking change is introduced

**Comment by matthewelwell** [comment]
> I need to make sure the EE can function as a single container

@dabeeeenster so, to get this to work, we'd need to write a new dockerfile in the enterprise repository (or finally tackle the disconnect between EE and core). Writing a new dockerfile wouldn't be too difficult but, before we do that, do we think that enterprise on-prem customers will actually _want_ to have the application running via the single image? Or, do we think that 99% would choose to run them separately anyway for more control over the scaling policies of each service?

**Comment by dabeeeenster** [comment]
Hmm - that's a good question! 

I think the best DX/UX is a simple setup. If customers then want to move to a more complex 2 service setup they should be able to do that, but having single image as default would be my preference. There could well be arguments against this tho!

**Comment by matthewelwell** [comment]
Yep, ok, I tend to agree. I'll have a look at adding the dockerfile to the enterprise repo.
